components/style/properties/properties.mako.rs

```diff
@@ -1145,7 +1145,7 @@ impl ComputedValues {
     #[inline]
     pub fn is_multicol(&self) -> bool {
         let style = self.get_column();
-        style.column_count.0.is_some() || style.column_width.0.is_some()
+        style.column_count.0.is_some() || style.column_width.is_first()
```

Comment:
I think your version is the equivalent of `&&`, not `||`.  But I get the idea.  Will do.